src/lib.rs

```diff
@@ -26,7 +26,22 @@ extern crate derivative;
 #[global_allocator]
 static ALLOC: tikv_jemallocator::Jemalloc = tikv_jemallocator::Jemalloc;
 
-#[cfg(all(feature = "tikv-jemallocator", feature = "allocation-tracing",))]
+#[cfg(all(
+    feature = "tikv-jemallocator",
+    feature = "allocation-tracing",
+    not(target_os = "macos")
+))]
+#[global_allocator]
+static ALLOC: self::internal_telemetry::allocations::Allocator<tikv_jemallocator::Jemalloc> =
+    self::internal_telemetry::allocations::get_grouped_tracing_allocator(
+        tikv_jemallocator::Jemalloc,
+    );
+
+#[cfg(all(
+    feature = "tikv-jemallocator",
+    feature = "allocation-tracing",
+    target_os = "macos"
+))]
```

Comment:
It looks like these are identical now, so I think we could collapse these two different blocks into a single one and just remove the `target_os = "macos"` portion?

Reply:
Ah yeah this can be greatly simplified now.